src/components/TodoList/TodoList.tsx

```diff
@@ -60,12 +71,12 @@ export const TodoList: React.FC<Props> = ({
         <select
           value={option}
           onChange={(event) => {
-            setOption(event.target.value);
+            setOption(+event.target.value);
```

Comment:
do you need `+` here?

Reply:
yes, I do 😅 
I asked you about the problem with typscript, it's not allowing to accept strings, because he is waiting for member of enum. This trick actually helped